Kitodo/src/main/java/org/kitodo/production/forms/createprocess/CreateProcessForm.java

```diff
@@ -478,4 +478,16 @@ private RulesetManagementInterface openRulesetFile(String fileName) throws IOExc
         }
         return ruleset;
     }
+
+    public void initializePocesses() {
+        try {
+            ProcessGenerator processGenerator = new ProcessGenerator();
+            processGenerator.generateProcess(template.getId(), project.getId());
+            this.processes = new LinkedList<>(Collections.singletonList(processGenerator.getGeneratedProcess()));
+        } catch (ProcessGenerationException e) {
+            logger.error(e.getLocalizedMessage());
```

Comment:
Error should be transfered to frontend, shouldn't it?

Reply:
In this case, there is no need to transfer the error to frontend.